introduction/architecture.md

```diff
@@ -72,12 +72,11 @@ operations when removing deleted data according to automated retention policies.
 The runtime can perform such operations by simply dropping chunks (internal
 tables), rather than deleting individual rows.
 
-
 ## Single Node vs. Clustering [](single-node-vs-clustering)
```

Comment:
Should this be single node vs multi node? Do we want to stick to the "clustering" terminology as an official term for our system or just use it loosely, e.g., "the servers of a distributed database are considered to be in the same 'cluster'"

Reply:
Yeah, I think you're right. Edited!